doc-content/enterprise-only/bpmn/bpmn-subprocesses-ref.adoc

```diff
@@ -25,14 +25,32 @@ An embedded subprocess encapsulates a part of the process. It must contain a sta
 
 An ad hoc subprocess or process contains a number of embedded inner activities and is intended to be executed with a more flexible ordering compared to the typical routing of processes. Unlike regular processes, an ad hoc subprocess does not contain a complete, structured BPMN2 diagram description, for example, from start event to end event. Instead, the ad hoc subprocess contains only activities, sequence flows, gateways, and intermediate events. An ad hoc subprocess can also contain data objects and data associations. The activities within the ad hoc subprocesses are not required to have incoming and outgoing sequence flows. However, you can specify sequence flows between some of the contained activities. When used, sequence flows provide the same ordering constraints as in a regular process. To have any meaning, intermediate events must have outgoing sequence flows and they can be triggered multiple times while the ad hoc subprocess is active.
 
-
 .Reusable subprocess
-Reusable subprocesses appear collapsed within the containing process.
+Reusable subprocesses appear collapsed within the parent process. To configure a reusable subprocess, select the parent process, click image:getting-started/diagram_properties.png[], and expand *Implementation/Execution*. Set the following properties:
+
+* *Called Element*: The ID of the subprocess that the activity calls and instantiates.
+* *Independent*: If selected, the subprocess is started as an independent process. If not selected, the active subprocess is canceled when the parent process is terminated.
+* *Abort Parent*: If selected, non-independent reusable subprocesses can abort the parent process when there is an error during the execution of the called process instance. For example, when there's an error when trying to invoke the subprocess or when the subprocess instance is aborted. This property is visible only when the *Independant* property is not selected. The following rules apply:
+** If the reusable subprocess is independent, *Abort parent* is not available.
+** If the reusable subprocess is not independent, *Abort parent* is available.
+* *Wait for completion*: If selected, the subprocess continues if the previous process completes or is aborted. This property is selected by default.
```

Comment:
"subprocess continues" can be confusing. It could be explained here, what exactly will continue in execution. If the property is set to true, _On exit action_ is not performed until the called sub-process instance is terminated. The execution of the "parent" process continues once the _On exit action_ is finished.
In the description there is used "child" process, but "previous" process should be used instead.

Reply:
Reworked as suggested.